helpers/sql/lib/opentelemetry/helpers/sql/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Helpers
+    module Sql
+      VERSION = '0.0.0'
```

Comment:
Will this bump itself when toys runs? I think we've been setting it to `'0.1.0'` on recent releases of brand new gems.

Reply:
Yes. The toys gem will bump this.